core/src/main/java/se/tre/freki/storage/Store.java

```diff
@@ -25,12 +25,23 @@
   //
   @Nonnull
   public abstract ListenableFuture<LabelId> createLabel(final String name,
-                                                          final LabelType type);
-
+                                                        final LabelType type);
+
+  /**
+   * Renames a old LabelId to the new {@code name}. This method makes no checks if this is a valid
+   * rename. The method calling this function needs to make sure this is done before. There are 2
+   * checks that needs to be done. Firstly we can not rename to a name that already exists. Secondly
+   * old {@code LabelId} needs to exist.
+   *
+   * @param name The new name.
+   * @param id The old label ID.
+   * @param type The label type.
+   * @return The LabelId that was used.
+   */
   @Nonnull
   public abstract ListenableFuture<LabelId> renameLabel(final String name,
```

Comment:
Can you rename the `String` name variable to `newName` while we're poking around with this code anyway?